parsers/IN_WE.py

```diff
@@ -87,10 +89,10 @@ def filter_raw_data(
     assert len(data) > 0
     assert kind != ""
 
-    dt_12_hour = arrow.get(target_datetime.strftime("%Y-%m-%d %I:%M")).datetime
+    dt_24_hour = arrow.get(target_datetime.strftime("%Y-%m-%d %H:%M")).datetime
```

Comment:
here as well, as well as updating the comment above about the 12 hour data format.

Reply:
Done